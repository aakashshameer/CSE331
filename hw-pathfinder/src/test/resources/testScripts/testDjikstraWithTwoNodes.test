#Find a path with the Dijkstra's algorithm with TwoNodes

CreateGraph twoNodesGraph

AddNode twoNodesGraph A
AddNode twoNodesGraph B

AddEdge twoNodesGraph A B 1.0
AddEdge twoNodesGraph A B 2.0

FindPath twoNodesGraph A B