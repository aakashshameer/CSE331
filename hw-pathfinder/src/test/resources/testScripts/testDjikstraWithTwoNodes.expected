#Find a path with the Dijkstra's algorithm with TwoNodes

created graph twoNodesGraph

added node A to twoNodesGraph
added node B to twoNodesGraph

added edge 1.000 from A to B in twoNodesGraph
added edge 2.000 from A to B in twoNodesGraph

path from A to B:
A to B with weight 1.000
total cost: 1.000